frontend/src/routes/CriticalQuestions/index.js

```diff
@@ -72,6 +72,18 @@ function CriticalQuestions(props) {
   const [contactCount, setContactCount] = useState(0);
   const [locationCount, setLocationCount] = useState(0);
 
+  let nextPage;
+  const { story } = useSelector((state) => state.story);
+  useEffect(() => {
+    if (!story) {
```

Comment:
what if there is no story because is the first time in the site? we should not go and fetch the story since there is nothing to fetch.
Is this being considered somewhere else in the codebase?

Reply:
This is covered by `fetchStory()` - essentially, we call the backend, it 404s, the `story` is set to `null` and that's it! This effect only runs when the `story` changes, so it's only fired once. The fields already have values because of the `formValues` default values, and those are never updated (because there's no story).